crates/l2/contracts/src/l1/CommonBridge.sol

```diff
@@ -118,7 +118,26 @@ contract CommonBridge is
         while (startingGas - gasleft() < amount) {}
     }
 
-    function _sendToL2(address from, SendValues memory sendValues) private {
+
+    bytes3 internal constant EIP7702_PREFIX = 0xef0100;
+    uint160 internal constant ADDRESS_ALIASING = uint160(0x1111000000000000000000000000000000001111);
```

Comment:
Let's add a comment saying this constant is the same Optimism uses, but that we could also choose another one if we wanted

Reply:
Let's actually use another constant, so our addresses don't end up the same as them (but let's keep the comment mentioning which value they use and that we chose another one)